src/Path.Transform.js

```diff
@@ -82,6 +82,16 @@ L.PathTransform.RotateHandle = L.PathTransform.Handle.extend({
 L.Handler.PathTransform = L.Handler.extend({
   options: {
     rotation: true,
+    showRotationAngle: true,
+    /**
+     * @param {number} angle
+     * @returns {string}
+     */
+    rotationAngleTooltipFormatter: function(angle) {
```

Comment:
maybe it would be better to rename this option, because we can use HTML to set tooltip content